src/modals/UnlockChunks/Overview.tsx

```diff
@@ -25,6 +28,14 @@ export const Overview = forwardRef(
     const { isFastUnstaking } = useUnstaking();
     const { t } = useTranslation('modals');
 
+    const { timeleft: eraTimeLeft } = useEraTimeLeft();
+    const { fromNow, timeleftAsString } = useTimeLeft({
+      refreshInterval: 60,
+      refreshCallback: () => eraTimeLeft,
+    });
+    const { durationSeconds } = useErasToTimeleft(bondDuration);
```

Comment:
When I tested the modal the text flickers between `12 hours` and `11 hours 59 mins` every so often. 

This value does not need to be a live, it is always 12 hours that needs to be statically converted from eras to time.